experiments/utils.py

```diff
@@ -17,9 +18,14 @@
 
 logger = logging.getLogger('experiments')
 
+redis = get_redis_client()
```

Comment:
If we can't contact Redis all of this will fail - be good to have some safeguarding and or logging

Reply:
Good point! I'll tweak.